exo-go/src/template_helpers/index.go

```diff
@@ -32,6 +33,8 @@ dependencies:
 services:
   public:
   private:
+
+templates:
```

Comment:
I think with the current setup we don't need to store information about the templates here any longer. The template names are the folder names in the `.exosphere` folder, the template URL is the Git remote there. We don't even need to know the URL any longer:
- exo add-template just adds the template if it doesn't exist and aborts with an error if it does
- exo fetch-templates just runs `git pull` in the submodules
- exo remove-template just deletes the Git submodule